packages/concerto-core/test/introspect/mapdeclaration.js

```diff
@@ -750,4 +750,12 @@ describe('MapDeclaration', () => {
             declaration.getValue().getParent().should.equal(declaration);
         });
     });
+
+    describe('#getNamespace', () => {
+        it('should return the correct namespace MapDeclaration Key and Value ', () => {
```

Comment:
should return the correct namespace MapDeclaration Key and Value
```suggestion
        it('should return the correct namespace for a Map Declaration Key and Value ', () => {
```

Reply:
Addressed.